### PR TITLE
Updated for AWS on ubuntu 16.04

### PR DESCRIPTION
**Pull request by brelloch**
This change is [Reviewable](https://reviewable.io/reviews/narrative-io/employee-handbook/4)

### CODE REVIEW

File: onboarding/workstation-setup/aws-tools.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ AWS Tools
   - Test that the config is OK
 
     ```bash
-    aws ec2 describe-instance
-    aws s3 ls s3://s3.amazonaws.com/narrative-artifact-releases
+    aws ec2 describe-instances
```
Comment (brelloch) [inline onboarding/workstation-setup/aws-tools.md]: @samidalouche Could you please verify these changes?

Comment (samidalouche) [inline onboarding/workstation-setup/aws-tools.md]: Yes, that's right, there was an error

File: onboarding/workstation-setup/Ubuntu 16.04/aws-tools.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+AWS Tools
```
Comment (samidalouche) [inline onboarding/workstation-setup/Ubuntu 16.04/aws-tools.md]: Would be nice to have this file under `onboarding/workstation-setup/aws-tools-ubuntu.md`. (I am unsure how spaces work with links)
The `onboarding/workstation-setup/aws-tools.md` file could be renamed to `aws-tools-macosx`, and the links in `workstation-setup` could be updated to link to both files depending on the OS. 

Obviously, "Mac OS X Workstation Setup" should be renamed to "Workstation Setup" . When I documented the the initial steps, I was expecting everyone to use Mac OS X, I guess ;-)

File: onboarding/workstation-setup/Ubuntu 16.04/aws-tools.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    ```bash
+    aws ec2 describe-instances
+    aws s3 ls s3://narrative-artifact-releases
+    ```
+  - Install [credstash](https://github.com/fugue/credstash)
+
+    > TODO: I had already installed pip months ago so I am not sure if the credstash install below is the normal ubuntu way
+
+    ```bash
+    sudo -H ~/.local/bin/pip install credstash
```
Comment (samidalouche) [inline onboarding/workstation-setup/Ubuntu 16.04/aws-tools.md]: is pip installed using `pyenv`? Wondering why we have to use sudo here.

Comment (brelloch) [inline onboarding/workstation-setup/Ubuntu 16.04/aws-tools.md]: Odds are it was not. I was helping some friends out about 6 months ago and have no idea how I installed a lot of what is running on here which is why I didn't document much.

**Comment by brelloch** [comment]
I don't think anyone is going to be using linux so I am just closing this.

**Pull request closed without merging**